packages/utils/src/parser/token.ts

```diff
@@ -9,6 +8,7 @@ import { FromStringToTokenError } from "../enums/token.enums";
  */
 export const convertStringToE8s = (
   value: string,
+  decimals = 8n,
```

Comment:
not sure what's 8n and 10n in this function.

also 8n is used twice in the module.

Can you extract (and `export const`) a constant?

I wont' be against a comment on why 8 and why 10.

Reply:
Good point, done!